README.md

```
@@ -1,16 +1,120 @@
+**Split payments automatically among multiple wallets with zero coding required.**
 
-| Package   | Description          | Version                       |
-| :-------- | :------------------- | :---------------------------- |
-| `Devnet`  | Devnet ui for Hydra  | https://hydra.metaplex.com?cluster=devnet |
-| `Mainnet` | Mainnet ui for Hydra | https://hydra.metaplex.com    |
+Hydra is a "fanout wallet" that lets you create a shared wallet where funds are automatically distributed to multiple recipients based on predetermined shares. Perfect for DAOs, creator royalties, team payments, and any scenario where you need to split money among multiple parties.
 
-![preview of claim page](https://github.com/user-attachments/assets/76d17dd5-b46b-4132-9eaf-4d25b133b959)
+## 🚀 Quick Start - Use the Hosted App
 
+**Mainnet:** https://hydra.metaplex.com  
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)